### ISSUE TITLE
PowerCommander 2fa KeeperSecurity.Utils.CryptoUtils namespace missing method

### ISSUE THREAD

**Issue by AmericanGeezus** [Keeper-Security/keeper-sdk-dotnet#25]
When attempting to retrieve 2fa code using the Powershell module I am receiving a method not found error.

```
 Get-KeeperRecords <UID> | 2fa

Method invocation failed because [KeeperSecurity.Utils.CryptoUtils] does not contain a method named 'GetTotpCode'.
At <userprofile>\Documents\WindowsPowerShell\Modules\PowerCommander\RecordCommands.ps1:195 char:4
+             [Tuple[string, int, int]]$code = [Utils.CryptoUtils]::Get ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : MethodNotFound
```

It appears to be a method accessibility/scope issue in the 2fa command. I am happy to provide more details in needed since I am sure the patch I worked out to RecordCommands.ps1 has implicating implications I haven't considered. 

The patched file (#24) works, so I believe it's helpful for any bug hunting if you guys decide this is an issue.

**Comment by sk-keeper** [comment]
Thank you very much for bringing up this issue.

`GetTotpCode` function was wrongfully made internal. 
It will be fixed shortly.

**Comment by sk-keeper** [comment]
A new release 0.9.9 has been published.
It addresses the issue with GetTotpCode method incorrect access modifier.

**Comment by AmericanGeezus** [comment]
Resolved with PowerCommander Release 0.9.9.
